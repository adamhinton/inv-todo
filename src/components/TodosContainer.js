//completed true false - passed in from App
// todoList and setTodoList passed in from app
// dynamic header
// filter over todoList looking for isCompleted = {isCompleted}
// checkbox : just completed boolean
// onClick:  just set completed to !completed. Update state
// delete button should be same

const TodosContainer = (props) => {
  return <></>;
};

export default TodosContainer;
