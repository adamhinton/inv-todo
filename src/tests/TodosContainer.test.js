import TodosContainer from "../components/TodosContainer";
import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
