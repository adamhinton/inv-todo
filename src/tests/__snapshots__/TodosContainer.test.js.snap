// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[4] TodosContainer Matches snapshot from 5.29.22 1`] = `
<section>
  <h2
    className="sc-gsnTZi cCbIxM"
  >
    Completed
  </h2>
  <ul>
    <li
      className="sc-bczRLJ bMzBMl"
      onClick={[Function]}
    >
      <input
        defaultChecked={true}
        type="checkbox"
      />
      <h4>
        Task one
      </h4>
    </li>
  </ul>
</section>
`;

exports[`[4] TodosContainer isCompleted=true matches snapshot from 5.29.22 1`] = `
<section>
  <h2
    className="sc-gsnTZi cCbIxM"
  >
    Completed
  </h2>
  <ul>
    <li
      className="sc-bczRLJ bMzBMl"
      onClick={[Function]}
    >
      <input
        defaultChecked={true}
        type="checkbox"
      />
      <h4>
        Task one
      </h4>
    </li>
  </ul>
</section>
`;
